src/entity/aggregator.rs

```diff
@@ -0,0 +1,114 @@
+use super::{account, membership, url::Url};
+use sea_orm::{entity::prelude::*, IntoActiveModel, Set};
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use std::fmt::{self, Display, Formatter};
+use std::str::FromStr;
+use time::OffsetDateTime;
+
+mod update_aggregator;
+pub use update_aggregator::UpdateAggregator;
+
+mod new_aggregator;
+pub use new_aggregator::NewAggregator;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, Serialize, Deserialize)]
+#[sea_orm(table_name = "aggregator")]
+pub struct Model {
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub id: Uuid,
+    pub account_id: Option<Uuid>,
```

Comment:
I think it's worthwhile to capture some of the doc strings provided in https://github.com/divviup/divviup-api/issues/196. Not all are necessary (id/created_at/updated_at/etc. is obvious what it is), but the nuances around account_id seem important to me.